bulk_course_settings/views.py

```diff
@@ -60,5 +59,18 @@ def form_valid(self, form):
         return response
 
 
-class BulkSettingsRevertView(LoginRequiredMixin,CreateView):
-    form_class = ""
+class BulkSettingsRevertView(LoginRequiredMixin, View):
+
+    def get(self, request, school_id, job_id):
+        related_bulk_job = BulkCourseSettingsJob.objects.get(id=job_id)
+        new_bulk_job = BulkCourseSettingsJob.objects.create(related_job_id=related_bulk_job.id,
+                                                            school_id=school_id,
+                                                            term_id=related_bulk_job.term_id,
+                                                            setting_to_be_modified=related_bulk_job.setting_to_be_modified)
+
+        utils.queue_bulk_settings_job(bulk_settings_id=new_bulk_job.id, school_id=school_id,
+                                      term_id=related_bulk_job.term_id,
+                                      setting_to_be_modified=related_bulk_job.setting_to_be_modified,
+                                      desired_setting='REVERT')
+
```

Comment:
This would be a good place to use the Django Messages framework to display a success message to the user at the top of the next page: https://docs.djangoproject.com/en/1.11/ref/contrib/messages/